[CSP] Revert PR 34921 (#35132)
demisto/content: Packs/Base/ReleaseNotes/1_34_19.md

#### Scripts

##### CommonServerPython

> Revert the version 1.34.15 due a bug in the **generic_http_request** function that he handled the files improperly.

Reviewer: ```suggestion
Revert version 1.34.15 due to an issue in **generic_http_request** causing improper handling of files.
```
